ledger/ledger-security-test-tool/BUILD.bazel

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//bazel_tools:scala.bzl", "da_scala_library", "da_scala_binary")
+
+da_scala_library(
+    name = "ledger-security-test-tool",
+    srcs = glob(["src/tool/scala/**/*.scala"]),
+    scala_deps = [
+        "@maven//:org_scalactic_scalactic",
+        "@maven//:org_scalatest_scalatest_core",
+        "@maven//:org_typelevel_cats_core",
+        "@maven//:org_typelevel_cats_kernel",
+        "@maven//:com_chuusai_shapeless",
+        "@maven//:io_circe_circe_core",
+        "@maven//:io_circe_circe_generic",
+        "@maven//:io_circe_circe_parser",
+        "@maven//:com_github_pathikrit_better_files",
+        "@maven//:com_lihaoyi_sourcecode",
+
+    ],
+    tags = ["maven_coordinates=com.daml:ledger-security-test-tool:__VERSION__"],
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "//ledger/ledger-security-test-lib",
```

Comment:
Can we have them in the same Bazel package? They are both pertaining to security evidencing, right?

Reply:
I think most of the test suite code will be then dependent on `ledger-security-test-lib`, but they dont need to be dependent on the tool code, but for simplification we could do that, yes...

Reply:
You can still independent targets but in the same package (dir)